package.json

```diff
@@ -64,5 +64,6 @@
     "extends": [
       "@commitlint/config-conventional"
     ]
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
This should be removed.

Reply:
Sorry, for some reason it gets automatically added when I run it locally.

Reply:
Pretty sure it's `corepack` doing this @Keyrxng rfc

Reply:
Correct it is indeed corepack